scripts/patterns-reorg/entries.py

```diff
@@ -0,0 +1,538 @@
+# This code is a Qiskit project.
```

Comment:
This file can be mostly skimmed. We'll work closely with the content team to finalize the page titles and new URLs. This file is the Source Of Truth™️ for the migration.